### PR TITLE
Added support for Python 3.13 (latest version) in tox.ini

### PR DESCRIPTION
**Pull request by mohsinsurani**
digit_group_separator and price is not none - make sured to make all the test passed.

### CODE REVIEW

**Comment by mohsinsurani** [comment]
Thanks. I added Support for different versions of python

**Comment by Gallaecio** [comment]
Minor thing: could you remove the spaces after the comma? For consistency with other projects.

**Comment by mohsinsurani** [comment]
No worries. I removed spaces.

**Comment by Gallaecio** [comment]
Thanks.

I think the only things left are to update these accordingly:

- https://github.com/scrapinghub/price-parser/blob/bfd1f14877400fc213a4172204e550463240af13/.github/workflows/main.yml#L25-L27
- https://github.com/scrapinghub/price-parser/blob/bfd1f14877400fc213a4172204e550463240af13/setup.py#L28

**Comment by mohsinsurani** [comment]
Thanks. I updated the code. Do I require to change for mypy?

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -25,10 +25,19 @@ jobs:
...
             TOXENV: py
+        - python-version: "3.11"
+          env:
+            TOXENV: py311
+        - python-version: "3.12"
+          env:
+            TOXENV: py312
+        - python-version: "3.13"
+          env:
+            TOXENV: py313
```
Comment (Gallaecio) [inline .github/workflows/main.yml:36]:
```suggestion
            TOXENV: py
        - python-version: "3.12"
          env:
            TOXENV: py
        - python-version: "3.13"
          env:
            TOXENV: py
```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -25,10 +25,19 @@ jobs:
...
         - python-version: "3.10"
           env:
             TOXENV: mypy
-
+
```
Comment (Gallaecio) [inline .github/workflows/main.yml:40]:
```suggestion

```

**Comment by Gallaecio** [comment]
> Do I require to change for mypy?

Great point. It should indeed run in the latest Python version ideally.

**Comment by mohsinsurani** [comment]
Thanks, I made the changes.

**Comment by Gallaecio** [comment]
https://github.com/scrapinghub/price-parser/pull/66/files#r1810447696 is not addressed yet. It’s empty space in line 40 of the new code.

**Comment by mohsinsurani** [comment]
Sorry, I missed it. I committed this change.

**Comment by Gallaecio** [comment]
I’ve just realized that I recently lost admin access to the repo :facepalm: 

As soon as I get it back I’ll approve the CI run, and merge if all goes well. Thanks!

**Pull request merged**